src/WebJobs.Extensions.DurableTask.Analyzers/Analyzers/EntityInterface/InterfaceAnalyzer.cs

```diff
@@ -44,21 +55,18 @@ public override void Initialize(AnalysisContext context)
 
         private void RegisterAnalyzers(CompilationAnalysisContext context)
         {
-            InterfaceContentAnalyzer contentAnalyzer = new InterfaceContentAnalyzer();
-            ParameterAnalyzer parameterAnalyzer = new ParameterAnalyzer();
-            EntityInterfaceReturnTypeAnalyzer returnTypeAnalyzer = new EntityInterfaceReturnTypeAnalyzer();
-
             foreach (EntityInterface entityInterface in entityInterfacesList)
             {
-                contentAnalyzer.ReportProblems(context, entityInterface);
-                parameterAnalyzer.ReportProblems(context, entityInterface);
-                returnTypeAnalyzer.ReportProblems(context, entityInterface);
+                InterfaceContentAnalyzer.ReportProblems(context, entityInterface);
+                ParameterAnalyzer.ReportProblems(context, entityInterface);
+                EntityInterfaceReturnTypeAnalyzer.ReportProblems(context, entityInterface);
             }
         }
 
         public void FindEntityCalls(SyntaxNodeAnalysisContext context)
         {
-            if (context.Node is MemberAccessExpressionSyntax expression)
+            var expression = context.Node as MemberAccessExpressionSyntax;
+            if (expression != null && SyntaxNodeUtils.IsInsideFunction(expression))
```

Comment:
You can simplify this back to 

`if (context.Node is MemberAccessExpressionSyntax expression && SyntaxNodeUtils.IsInsideFunction(expression))`